packages/ragbits-document-search/src/ragbits/document_search/_main.py

```diff
@@ -163,5 +164,16 @@ async def insert_elements(self, elements: list[Element]) -> None:
             elements: The list of Elements to insert.
         """
         vectors = await self.embedder.embed_text([element.get_key() for element in elements])
+
+        image_elements = [element for element in elements if isinstance(element, ImageElement)]
```

Comment:
What about the previous case support for description of images? In this case we would include image elements twice (one with the upper loop over all elements and second with this loop). Maybe we should add some parameter that would allow for method selection?

Reply:
> What about the previous case support for description of images?

I wanted to preserve the previous method, that's why I don't filter out image elements when creating the `vectors` list. You cannot have one embedding that covers both the text content and the image content, so it seemed reasonable to me to use two vectors to cover both.

I'm open to making this configurable, but maybe this configuration should be on the level of document processor - because running OCR and ignoring it during ingestion would be pretty wasteful

Reply:
Sounds good to me, we can discuss it with the broader group as well. Feel free to add a separate task for it and merge this one or add it to this already existing PR.

Reply:
@konrad-czarnota-ds

I created https://github.com/deepsense-ai/ragbits/issues/175

Looks ok?

Reply:
Looks good